### DISCUSSION TITLE
A different app icon for ReVanced YouTube app

### DISCUSSION THREAD

**Discussion by Hexaae** [ReVanced/revanced-discussions#1004] [General]
I noticed that ReVanced YouTube has the same icon of Manager.
Can you use different icons for user-friendness?

**Comment by oSumAtrIX** [comment]
No, the ReVanced logo is the default/ placeholder for the branding patch that will eventually be lifted off its constraints to YouTube, so a default one has to be present.

**Comment by Hexaae** [comment]
yeah, but can't the patches add different icon to the newly generated ReVanced apps? Or maybe an icon selection to choose from when you choose to patch an APK... Having multiple apps with the same identical icon in the launcher can be annoying and you have to read the app name below.

**Reply by oSumAtrIX** [reply]
It is the default icon. You can either chose your own one, use the default one, or disable the patch.

**Reply by Hexaae** [reply]
Ah, so I simply missed that option and you're saying you can actually already choose the (new) icon before patching APK?
I can see only a Custom Branding patch yes/no, but how do I select my custom launcher icon at this point? As far as I can see you can't choose your own.

**Reply by Hexaae** [reply]
@oSumAtrIX
Given info is misleading. You can't actually choose your own icon ATM when patching an APK from ReVanced Manager. There are only two options: use original APK icon or replace it with default ReVanced Manager icon.
Where can I post a feature request?

**Reply by oSumAtrIX** [reply]
Depends on the request for which project.

**Reply by winterhub01** [reply]
Where can I find this option please?
